Dockerfile-curl.template

```
@@ -3,6 +3,7 @@ FROM {{ env.dist }}:{{ env.codename }}
 RUN set -eux; \
 	apt-get update; \
 	apt-get install -y --no-install-recommends \
+		apt-transport-https \
```

I'm generally OK with this addition (since it fits the conceptual "spirit" of what the `curl` variant is expected to include), but it's only necessary in `debian:stretch` and `ubuntu:xenial` (see https://packages.debian.org/search?keywords=apt-transport-https and https://packages.ubuntu.com/search?keywords=apt-transport-https), on `debian:buster` and above and `ubuntu:bionic` and above it is merely a transitional package ("It can be safely removed."), so I'd propose the following change to the template:

```suggestion
{{ if [ "stretch", "xenial" ] | index(env.dist) then ( -}}
		apt-transport-https \
{{ ) else "" end -}}
```

(This should also decrease the number of modified files from 10 to 3. :smile: :+1:)

---

Fix included 👍🏻